### DISCUSSION TITLE
Issue installing Frigate

### DISCUSSION THREAD

**Discussion by Snackington** [tteck/Proxmox#2829] [Help Needed]
Tied it twice, both times using the default options, it gets almost done then gives this. Granted it does eventually finish installing. Any ideas what I'm doing wrong?

```
`✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Installed Python3 Dependencies
 ✓ Installed Node.js
 ✓ Installed go2rtc
 ✓ Set Up Hardware Acceleration
 - Installing Frigate v0.13.2 (Perseverance)  \/dev/stdin: line 51: 23583 Segmentation fault      sleep 0.2
/dev/stdin: line 37: 23584 Segmentation fault      sleep 0.2

[ERROR] in line 57: exit code 0: while executing command sleep 0.2`
```

**Comment by 36grad** [comment]
Had the error message, twice actually, it did work though afterwards.
However, since I was not able to enable the iGPU in the BIOS, I installed again on a different machine, there I did not have any error messages.
